pls_balances/scripts/template.py

```diff
@@ -0,0 +1,19 @@
+import pandas as pd
+import numpy as np
+from biom import Table, load_table
+
+metadata = pd.read_table("pls_balances/data/globalpatterns/GlobalPatterns_mapping.txt", index_col=0)
+
+table = load_table("pls_balances/data/globalpatterns/globalpatterns_s1_x13_filter.biom")
```

Comment:
Will need to have hardcoded absolute paths.  This an be done with

```
curpath = os.path.abpath(__file__)
table_file = "pls_balances/data/globalpatterns/globalpatterns_s1_x13_filter.biom"
table = load_table('%s/%s' (curpath, table_file)
```

Reply:
Actually don't worry about this -- we can add this as a parameter in the CLI.

I was imagining that the templates can be added to all of the functions that we have coded up in [generate.py](https://github.com/knightlab-analyses/pls-balances/blob/master/pls_balances/scripts/generate.py)


If we take [compositional_effect_size](https://github.com/knightlab-analyses/pls-balances/blob/master/pls_balances/scripts/generate.py#L89) as an example, I was thinking that you could add the following two decorators
```
@click.option('--template-biom', default=None,
              help='Template biom file path.')
@click.option('--template-sample-name', default=None,
              help='Template sample name.')
```

Then you can extract the corresponding template vector, and resize it to match the other user specified parameters.  This is just one option of many.

But once we have a working `generate.py` in place, we can autogenerate all of the testing files prior to the actual snakemake run (i.e. you don't need snakemake for this step to work).

I'd look at `launch.sh` for some examples of how these generator scripts should work.